test/setup_script.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Do we need to add license to this file?
Can we add our setup steps to this file?

Reply:
I will add a licence in this file.
Yes, setup steps can be added in this file. But since not setup steps are required for our tests, the file is empty.